### DISCUSSION TITLE
OP-PROPOSER

### DISCUSSION THREAD

**Discussion by AbhishekCDN** [ethereum-optimism/developers#522] [Application Development] [answered]
Hello, We have run our own l2 chain on local. When I run proposer, the proposer node is started but it did not make any transactions. Batcher, Geth, Node is working fine but proposer is not working.

**Comment by ajsutton** [comment, marked as answer]
This isn't an issue with the specs so would be better asked in https://github.com/ethereum-optimism/developers/discussions/

**Comment by opfocus** [comment]
Does OP-PROPOSER have logs to remind you of existing problems?
